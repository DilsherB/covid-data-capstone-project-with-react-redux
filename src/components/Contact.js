import React from "react";
import ContactBanner from "./ContactBanner";

const Contact = () => {
  return (
    <div className="py-3 px-10 text-left">
      <ContactBanner />
      <p>
        I am a full-stack Software Engineer. I have done handfull projects at
        &ldquo;Microverse&ldquo;. It gives me confidence and a lot of
        experience. During this journey at &ldquo;Microverse&ldquo; I have
        learned HTML, CSS, BootStrap, TailwindCSS, JavaScript, React js,
        MongoDB, Ruby and Ruby on Rails.
      </p>
      <br />
      <p>
        Through my day-to-day job, I&apos;ve developed extensive knowledge in
        the following
      </p>
      <br />
      <ul className="space-y-1 list-disc list-inside indent-5">
        <li>
          <span className="font-bold">Front-End: </span>JavaScript, React,
          Redux, Next Js, HTML5, CSS3, SASS, MaterialUI | Bootstrap |
          TailwindCSS
        </li>
        <li>
          <span className="font-bold">Back-End: </span>Node Js, Express, Ruby,
          Rails, MongoDB, MySQL, Postgre
        </li>
        <li>
          <span className="font-bold">Tools: </span>Git, GitHub
        </li>
        <li>
          <span className="font-bold">Prototyping: </span>Figma, Adobe XD,
          Sketch
        </li>
        <li>
          <span className="font-bold">Professional: </span>Remote
          Pair-Programming, collaboration, code review, mentoring, effective
          communication skills across cultures
        </li>
      </ul>
      <br />
      <p>
        My mission is to use my expertise to empower other people&apos;s dreams.
      </p>
      <br />
      <p>Let&apos;s build a better world using code. One keypress at a time!</p>
      <br />
      <p>
        I am currently looking for my next job in Software Development. If
        you&apos;d like to get in contact about potential job opportunities,
        please send me a message or email at dilsherb@gmail.com
      </p>
    </div>
  );
};

export default Contact;
