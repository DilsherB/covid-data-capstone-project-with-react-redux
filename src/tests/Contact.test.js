// eslint-disable-next-line import/no-extraneous-dependencies
import renderer from "react-test-renderer";
import React from "react";
import Contact from "../components/Contact";

const renderTree = (tree) => renderer.create(tree);
describe("<Contact>", () => {
  it("should render component", () => {
    expect(renderTree(<Contact />).toJSON()).toMatchSnapshot();
  });
});
