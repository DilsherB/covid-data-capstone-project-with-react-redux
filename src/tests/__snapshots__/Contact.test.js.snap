// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contact> should render component 1`] = `
<div
  className="py-3 px-10 text-left"
>
  <div
    className="bg-slate-300 rounded-lg p-3 flex gap-6 mb-5"
  >
    <img
      alt="Developer"
      className="rounded-full w-28 h-28"
      src="My Profile Photo.jpeg"
    />
    <div>
      <h1
        className="text-2xl font-bold"
      >
        Dilsher Balouch
      </h1>
      <p
        className="text-lg font-semibold text-slate-600"
      >
        Full-Stack Developer
      </p>
      <p
        className="text-lg font-semibold text-slate-600 flex justify-between"
      >
        <a
          href="https://www.linkedin.com/in/dilsherbalouch/"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
            />
          </svg>
        </a>
        <a
          href="https://www.facebook.com/dilsherb/"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M504 256C504 119 393 8 256 8S8 119 8 256c0 123.78 90.69 226.38 209.25 245V327.69h-63V256h63v-54.64c0-62.15 37-96.48 93.67-96.48 27.14 0 55.52 4.84 55.52 4.84v61h-31.28c-30.8 0-40.41 19.12-40.41 38.73V256h68.78l-11 71.69h-57.78V501C413.31 482.38 504 379.78 504 256z"
            />
          </svg>
        </a>
        <a
          href="https://github.com/DilsherB"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 496 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            />
          </svg>
        </a>
        <a
          href="https://stackoverflow.com/users/17731460/dilsher-balouch"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 384 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M290.7 311L95 269.7 86.8 309l195.7 41zm51-87L188.2 95.7l-25.5 30.8 153.5 128.3zm-31.2 39.7L129.2 179l-16.7 36.5L293.7 300zM262 32l-32 24 119.3 160.3 32-24zm20.5 328h-200v39.7h200zm39.7 80H42.7V320h-40v160h359.5V320h-40z"
            />
          </svg>
        </a>
      </p>
    </div>
  </div>
  <p>
    I am a full-stack Software Engineer. I have done handfull projects at “Microverse“. It gives me confidence and a lot of experience. During this journey at “Microverse“ I have learned HTML, CSS, BootStrap, TailwindCSS, JavaScript, React js, MongoDB, Ruby and Ruby on Rails.
  </p>
  <br />
  <p>
    Through my day-to-day job, I've developed extensive knowledge in the following
  </p>
  <br />
  <ul
    className="space-y-1 list-disc list-inside indent-5"
  >
    <li>
      <span
        className="font-bold"
      >
        Front-End: 
      </span>
      JavaScript, React, Redux, Next Js, HTML5, CSS3, SASS, MaterialUI | Bootstrap | TailwindCSS
    </li>
    <li>
      <span
        className="font-bold"
      >
        Back-End: 
      </span>
      Node Js, Express, Ruby, Rails, MongoDB, MySQL, Postgre
    </li>
    <li>
      <span
        className="font-bold"
      >
        Tools: 
      </span>
      Git, GitHub
    </li>
    <li>
      <span
        className="font-bold"
      >
        Prototyping: 
      </span>
      Figma, Adobe XD, Sketch
    </li>
    <li>
      <span
        className="font-bold"
      >
        Professional: 
      </span>
      Remote Pair-Programming, collaboration, code review, mentoring, effective communication skills across cultures
    </li>
  </ul>
  <br />
  <p>
    My mission is to use my expertise to empower other people's dreams.
  </p>
  <br />
  <p>
    Let's build a better world using code. One keypress at a time!
  </p>
  <br />
  <p>
    I am currently looking for my next job in Software Development. If you'd like to get in contact about potential job opportunities, please send me a message or email at dilsherb@gmail.com
  </p>
</div>
`;
